linearSearch/linearSearch.py

```diff
@@ -0,0 +1,7 @@
+def linearSearch(arr, item):
+  for i in range(0, len(arr)):
+    if (arr[i] == item):
+      return i
+  return -1
+
+linearSearch([1, 2, 3, 5, 7, 11, 13, 17, 23, 31], 13)
```

Comment:
Please change to this line
```suggestion
print("Element found at index "+str(linearSearch([1, 2, 3, 5, 7, 11, 13, 17, 23, 31], 13)))
```

Reply:
Ok, sure. I'll update the code.